crates/connectors/ndc-postgres/src/schema.rs

```diff
@@ -538,6 +538,15 @@ fn experimental_insert_to_procedure(
             description: None,
         },
     );
+    arguments.insert(
+        insert.constraint.argument_name.clone(),
+        models::ArgumentInfo {
+            argument_type: models::Type::Predicate {
+                object_type_name: insert.collection_name.clone(),
+            },
+            description: Some(insert.constraint.description.clone()),
+        },
+    );
```

Comment:
add the predicate to the schema of experimental/insert.

Reply:
Should these be optional / nullable? Otherwise if the user doesn't want to use permissions they have to explicitly add a `const true`-esq expression to their engine metadata, which seems like a pain.

Reply:
I assumed the standard usecase is to apply permissions to insert always, but I guess we could? Let's not do this as part of this PR though, and do that for `delete` as well.

Reply:
Yeah, I'm fine with that, we'd be weakening the contract so it wouldn't be a breaking change, so no rush.

Reply:
This is the experimental mutations version, we can break it as much as we like :-)